app/controllers/openid_application_controller.rb

```diff
@@ -198,6 +215,27 @@ def fetch_smart_launch_context
     raise error_klass('Invalid launch context')
   end
 
+  def fetch_issued(token_string)
+    return nil unless Settings.oidc.issued_url
+
+    response = RestClient.get(Settings.oidc.issued_url,
+                              { Authorization: 'Bearer ' + token_string })
+    raise error_klass('Invalid token') if response.nil?
+
+    if response.code == 200
+      json_response = JSON.parse(response.body)
+      if json_response['scopes']
+        json_response['scp'] = json_response['scopes']
+        json_response['scopes'] = nil
+      end
+      json_response
+    end
+  rescue => e
+    raise raise error_klass('Invalid token') if e.to_s.include?('Unauthorized')
```

Comment:
Is `raise raise` correct or a typo?

Reply:
Typo, should just be a single raise.  Will correct.